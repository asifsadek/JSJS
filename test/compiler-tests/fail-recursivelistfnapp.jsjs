val f = /\(x) => x([x, x]);
