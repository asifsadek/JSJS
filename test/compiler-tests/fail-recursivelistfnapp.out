Type error: expected value of type 'A', got a value of type '(list A) -> B' instead
